docs/pages/careers/developer-advocate.md

```diff
@@ -1,116 +1,111 @@
-- **Relationship-building**.
-  - Create relationships with fellow ecosystem influencers and open-source leaders.
-  - Nurture ongoing connections with user group members to build personal relationships and deeply understand their needs, usage, journeys, and barriers to adoption.
-- **Technical**.
-  - Write code where required to support how-to content, blog posts, and presentations.
-  - Create inspiring sample applications, documentation, and tutorials.
-  - Give feedback to product management, to influence the product roadmaps based on developers' needs.
-  - Follow GitHub issues to understand where developers face frustration and develop strategies to overcome these. This could be suggesting or implementing documentation updates, or proposing or contributing code changes that solve the core issue.
+### Key responsibilities
 
-For the right candidate:
+- Create educational materials (tutorials, how-to guides, demo apps) for devs of all skill levels
+- Seek out and implement improvements to the documentation
+- Contribute to the docs infrastructure and UX
+- Engage with the community and gather feedback for product teams
+- Serve as the "voice" of the company through social media, conferences, and events
 
-- Working with the Leadership to construct and execute on a hiring plan to grow the Developer Experience team.
+### Tasks and initiatives you might take on
 
-### Here are a few initiatives you might work on
+- Create tutorials for migrating from competitor products
+- Improve the Algolia search experience
+- Be a first responder for GitHub issues and PRs related to docs
+- Build example apps with MUI products and create how-to content around them
+- Research and write case studies
+- Overhaul the [Showcase](https://mui.com/material-ui/discover-more/showcase/)
+- Revamp the company blog infrastructure to empower less technical teammates to contribute
+- Solicit (and potentially contribute to) integrations with other popular libraries and frameworks
 
-- Create a learning section in the documentation for hybrid written and video tutorials.
-- Own the Algolia search experience, improving the content on the most frequent search queries.
-- Take ownership of the company's Twitter account.
-- Level up the showcase, cherry-picking the best websites and apps of the community.
-- Revamp the blog post infrastructure to empower the rest of the team to create more content.
-- Start a monthly newsletter to engage with the community.
-- Organize a MUIConf (once we go >50 people)
-- Think about creative ways to promote Base UI, Joy UI, MUI X, MUI X Pro, and new products.
+## Who we're looking for
 
-## About you
+You're a React developer with a few years of professional experience on the front end, and maybe you've worked with design systems before.
```

Comment:
This feels a bit duplicate with what comes next

Reply:
agreed, feels redundant - removed